src/manager.js

```diff
@@ -777,7 +777,18 @@ AdManager.prototype.refreshSlots = function (slotsToLoad) {
     return;
   }
 
-  this.googletag.pubads().updateCorrelator();
+  // If any of the ads are eager loaded, don't update the correlator
+  var updateCorrelator = true;
+  for (var i = 0; i < slotsToLoad.length; i++) {
+    var thisSlot = slotsToLoad[i];
+    if (thisSlot.eagerLoad) {
```

Comment:
probably could just inline this and check `if (slotsToLoad[i].eagerLoad) {` since you're not using that variable anywhere else.  could also `break` out of the for loop once you find one